async_simple/coro/test/LazyTest.cpp

```diff
@@ -420,7 +420,7 @@ TEST_F(LazyTest, testCollectAll) {
 TEST_F(LazyTest, testCollectAllBatched) {
     size_t task_num = 5000;
     long total = 0;
-    for (auto i = 0; i < task_num; i++) {
+    for (int i = 0; i < static_cast<int>(task_num); i++) {
```

Comment:
Is this change necessary?

Reply:
ping?

Reply:
now task_num from `size_t` to  `int`, static_cast is unnecessary.